cli/tools/init/mod.rs

```diff
@@ -40,6 +40,15 @@ pub async fn init_project(init_flags: InitFlags) -> Result<(), AnyError> {
     .replace("{CURRENT_STD_URL}", deno_std::CURRENT_STD_URL.as_str());
   create_file(&dir, "main_test.ts", &main_test_ts)?;
 
+  create_file(&dir, "deno.json", include_str!("./templates/deno.json"))?;
+  let vscode_dir = dir.join(".vscode");
+  std::fs::create_dir_all(vscode_dir)?;
+  create_file(
```

Comment:
Bit of a bonus, but it would be neat if this modified the file to enable deno, if it existed.

Reply:
We already prompt if there are any files in the directory, is it needed then?

Reply:
Nope. Maybe something for the future.